pipeline.py

```diff
@@ -59,30 +82,52 @@ class TextProcessingPipeline:
     Process articles from corpus manager
     """
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
+        self._text = ''
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for file in self.corpus_manager.get_articles().values():
+            self._text = file.get_raw_text()
+            file.save_processed(' '.join(self._process()))
```

Comment:
I am sure you will see a problem here as _process should return the list of tokens.